projects/bundle-test/tsconfig.app.json

```diff
@@ -6,7 +6,7 @@
     "types": [],
     "paths": {
         "igniteui-angular": [
-          "projects/igniteui-angular/src/public_api.ts"
+          "dist/igniteui-angular"
```

Comment:
Should this path change to `dist/`? I don't think we have to build the product in order to test.

Reply:
See last failed build, didn't quite take. Those internal refs don't get treated the same as a package, more like parts of the same app source. Same already happened with the i18n package where it _was_ tree-shaking without the fix when linking the source.
Also makes sense - we're interested in the shipped product and that is the dist output.